content/arduino-cloud/01.getting-started/06.dashboard-widgets/dashboard-widgets.md

```diff
@@ -126,6 +126,22 @@ else if(stepperVariable == 11){
 Serial.println(stepperVariable);
 ```
 
+### Time Picker
+
+![Time Picker Widget](assets/time-picker.png)
+
+The time picker widget is used to retrieve the local time on your board.
```

Comment:
```suggestion
The time picker widget is used to represent and set a time and/or a date.
```

Reply:
It represents whatever time the variable contains, then it's up to you what time you want to put in the variable :)